modules/otel-config-instrumentation.adoc

```diff
@@ -98,6 +98,94 @@ spec:
 
 |===
 
+[id="otel-instrumentation-options_exporter_{context}"]
+== Exporter configuration
+
+Although the `Instrumentation` custom resource supports setting up one or more exporters per signal, the auto-instrumentation configures only the OTLP Exporter. Due to this, you must configure the endpoint to point to the OTLP Receiver on the Collector. You can do so by using a config map as well as a Secret for the TLS and CA certificates.
+
+.Sample exporter configuration using a config map
+[source,yaml]
+----
+  exporter:
+    endpoint: https://production-collector.observability.svc.cluster.local:4317  # <1>
+    tls:
+      configMapName: ca-bundle  # <2>
+      ca_file: service-ca.crt # <3>
+----
+<1> Specifies the OpenTelemetry Protocol (OTLP) endpoint. The HTTPS scheme enables secure connection and requires a `tls` configuration.
+<2> Specifies the name of the config map. The config map must already exist in the namespace of the pod injecting the auto-instrumentation.
+<3> Points to the certificate file in the config map or the absolute path to the certificate if the certificate is already present in the workload file system.
+
+.Sample exporter configuration using a Secret
+[source,yaml]
+----
+  exporter:
+    endpoint: https://production-collector.observability.svc.cluster.local:4317  # <1>
+    tls:
+      secretName: serving-certs # <2>
+      ca_file: service-ca.crt # <3>
+      cert_file: tls.crt # <4>
+      key_file: tls.key # <5>
+----
+<1> Specifies the OpenTelemetry Protocol (OTLP) endpoint. The HTTPS scheme enables secure connection and requires a `tls` configuration.
+<2> Specifies the name of the Secret that contains a TLS certificate and key. The Secret must already exist in the namespace of the pod injecting the auto-instrumentation.
```

Comment:
TLS certificate, CA bundle and key. 

in this example the CA, TLS cert and key come from the secret